plugin/meta_data_store.go

```diff
@@ -0,0 +1,53 @@
+package plugin
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+)
+
+type metaDataStore struct {
+	dir           string
+	installTarget *installTarget
+}
+
+var errDisaleMetaDataStore = errors.New("MetaData disabled. could not detect owner/repo")
+
+func newMetaDataStore(pluginDir string, it *installTarget) (*metaDataStore, error) {
```

Comment:
[nit] IMO the variable name `target` is better than `it` but leave _it_ to you.

Reply:
fixed in fc3b81d3190e60262635d9a787670c6e0eb8a84b